### ISSUE TITLE
Themes - what would you like to do?

### ISSUE THREAD

**Issue by Phuket2** [omz/Pythonista-Issues#195]
@omz,  there appear to be a lot issues when working with themes or possibly lack of understanding on how to use themes.  I am guessing you are aware of that.  It seems you have a lot on your plate at the moment.  Would you prefer we hold on reporting theme issues for now?  It seems to me that until you get some time to give it a once over yourself there is not much point for us digging into it now.  

Maybe I am wrong about this.  Maybe I am not using the way it was intended to be used, but to me it does not seem ready for prime time yet.  

Eg. I just created a ui.TextView in a view that is presented using editor.present_themed.  The background of the tv is white and the text is a very light color.  I can change the bg_color of the tv view, but if I change the text_color  property it is set correctly, but the tv view sticks to displaying the very light color.  I also tried playing with editor.apply_ui_theme, I could not improve the situation.  

But I have see a lot of gotchas when working with themes.  I understand it's a lot more complicated than it looks on the surface.

Anyway, I understand there are a lot of higher priority items than themes.  I am just trying to be realistic about this and not flood your mind with trivial stuff.

So, in short , I am asking would you prefer we hold off reporting on/testing themes , or prefer we did.

Edit...
Btw, guessing you are using -1 in the red component to know to when to draw the theme color

**Comment by zrzka** [comment]
It seems to me that it contains lot of questions, not really a bug, no answers, going to close this one. Ian, if you still encounter theming problems, please, file new issue(s) and try to be specific what and when it doesn't work, etc. Thanks.
